microsoft-edge/devtools-guide-chromium/workspaces/index.md

```diff
@@ -176,69 +199,68 @@ The DOM tree is not the HTML source tagging.
 
-1. Navigate to the **Sources** tool.
+1. Click the **Sources** tab.
 
-1. In the **Navigator** pane (on the left), click the **Page** tab.
+1. In the **Navigator** pane, click the **Page** tab beside **Filesystem**. If the **Page** tab isn't showing, click `>>` (More tabs) and then select **Page**.
 
 1. Click **(index)**.  The HTML for the page opens.
 
-1. Replace `<h1>Workspaces Demo</h1>` with `<h1>I ❤️  Cake</h1>`.  Review the following figure.
+1. Replace `<h1>Workspaces Demo</h1>` with `<h1>I Love Cake</h1>`.  The demo title changes.
 
 1. Press `Ctrl`+`S` (Windows, Linux) or `Command`+`S` (macOS) to save the change.
 
 1. Refresh the page.  The `<h1>` element continues to display the new text after the page is refreshed.
 
    ![Change HTML from the Sources tool.](../media/workspaces-workspaces-demo-sources-page-h1.msft.png)
 
-1. Open `~/Desktop/app/index.html`.  The `<h1>` element contains the new text.
+1. Open `~/Desktop/app/index.html` in a text editor.  The `<h1>` element contains the new text.
 
 
 <!-- ====================================================================== -->
-## Step 4: Save a JavaScript change to disk
+#### Save a JavaScript change to disk
 
 The main place to use the code editor of DevTools is the **Sources** tool.  But sometimes you need to access other tools, such as the **Elements** tool or the **Console** panel, while editing files.  The **Quick source** tool gives you just the editor from the **Sources** tool, while any tool is open.
 
 To open the DevTools code editor alongside other tools:
 
-1. Navigate to the **Elements** tool.
+1. Select the **Elements** tool.
 
-1. Press `Ctrl`+`Shift`+`P` (Windows, Linux) or `Command`+`Shift`+`P` (macOS).  The **Command Menu** opens.
+1. Press `Ctrl`+`Shift`+`P` (Windows, Linux) or `Command`+`Shift`+`P` (macOS) to open the **Command Menu**.
 
-1. Type `quick`, and then select **Show Quick source**.  At the bottom of the DevTools window, the **Quick source** tool appears, displaying the contents of `index.html`, which is the last file you edited in the **Sources** tool.
+1. Type `quick` at the **Run** prompt, and then select **Show Quick source**.  At the bottom of the DevTools window, the **Quick source** tool appears, displaying the contents of `index.html`, which is the last file you edited in the **Sources** tool.
 
    ![Open the 'Quick source' tool by using the Command Menu.](../media/workspaces-workspaces-demo-search-show-quick-source.msft.png)
 
-1. Press `Ctrl`+`P` (Windows, Linux) or `Command`+`P` (macOS) to open the **Open File** dialog, as shown below.
+1. Press `Ctrl`+`P` (Windows, Linux) or `Command`+`P` (macOS) to open the **Open File** dialog, shown in the next screenshot.
 
-1. Type `script`, then select **app/script.js**.
+1. Type `script` at the **Open** prompt, then select **app/script.js**.
 
    ![Open script.js using the Open File dialog.](../media/workspaces-workspaces-demo-search-script.msft.png)
```

Comment:
```suggestion
   ![Opening script.js using the Open File dialog](../media/workspaces-workspaces-demo-search-script.msft.png)
```
d/k what dialog title for Linux, Windows, & Mac.  alt text requirement is pretty easy to meet, literally describe the particular screenshot.